pkg/cmd/server/server.go

```diff
@@ -456,6 +470,7 @@ var defaultRestorePriorities = []string{
 	"limitranges",
 	"pods",
 	"replicaset",
+	"customresourcedefinitions",
```

Comment:
This has already made it into `master` so can be removed from this PR

Reply:
removed